integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -4148,17 +4148,14 @@ msgid "Overview of the text understandability of the pages"
 msgstr "Übersicht der Textverständlichkeiten der Seiten"
 
 #: cms/templates/analytics/_page_hix_widget.html
-#, python-format
-msgid ""
-"The list shows the first %(n)s pages with the lowest HIX value in ascending "
-"order."
+msgid "The list shows the pages with the lowest HIX value in ascending order."
 msgstr ""
-"Die Liste zeigt die ersten %(n)s Seiten mit dem niedrigsten HIX-Wert "
-"aufsteigend sortiert."
+"Die Liste zeigt alle Seiten in aufsteigender Reihenfolge nach ihrem HIX Wert "
```

Comment:
```suggestion
"Die Liste zeigt alle Seiten in aufsteigender Reihenfolge nach ihrem HIX-Wert "
```
While in English, compound nouns are separated by a whitespace, [in German they are written in one word](https://www.duden.de/sprachwissen/sprachratgeber/Zusammengesetzte-Substantive), and can only be separated by a hyphen.